src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -5,11 +5,9 @@ import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.backend.{Backend, JobKey}
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.{CallKey, ExecutionStoreKey, OutputKey}
-import cromwell.engine.{SymbolStoreEntry, WorkflowDescriptor, WorkflowId, WorkflowState}
+import cromwell.engine.{SymbolStoreEntry, WorkflowDescriptor, WorkflowId, WorkflowState, _}
```

Comment:
Yeah, strangely the optimizer missed this one. :confused: I assume that due to scala's package objects, the verbose and short alternatives could technically mean something slightly different, but that didn't seem to apply in this case.